opm/common/utility/Serializer.hpp

```diff
@@ -545,53 +549,47 @@ class Serializer {
         T, std::void_t<decltype(std::declval<T>().serializeOp(std::declval<Serializer<Packer>&>()))>
     > : public std::true_type {};
 
-    //! \brief Handler for smart pointers.
+    //! \brief Handler for shared pointers.
     template<class PtrType>
-    void ptr(const PtrType& data)
+    void shared_ptr(const PtrType& data)
     {
         using T1 = typename PtrType::element_type;
-        void* data_ptr = reinterpret_cast<void*>(data.get());
+        uintptr_t data_ptr = reinterpret_cast<uintptr_t>(data.get());
```

Comment:
Tiny nit: Using `uintptr_t` means you need to `#include <stdint.h>`.  Optionally, you could use `std::uintptr_t` in which case you'd need to `#include <cstdint>`.

Reply:
yeah, my bad. used the std:: prefixed one.